### DISCUSSION TITLE
Welcome to PiIR Discussions!

### DISCUSSION THREAD

**Discussion by ts1** [ts1/PiIR#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by olegthegreat** [comment]
When running the command, I'm getting this, can someone please help:
)
![2024-12-11_18h04_56](https://github.com/user-attachments/assets/51d705bb-ae9b-4a91-88e5-6787928abd07)

**Reply by ts1** [reply]
Your Python version is outdated. Using underscores in numeric literals is supported in version 3.6 and above.
